src/components/ApplicationForm/Skills.js

```diff
@@ -28,28 +28,34 @@ const FormRow = ({ id, children }) => (
   </div>
 )
 
+const StyledTextArea = styled(TextArea)`
+  margin: 1em 0;
+`
+
 export default ({ formInputs, onChange, role, handleResume }) => {
   return (
     <>
-      <CenteredH1>
-        Flex your skills!{' '}
-        <span role="img" aria-label="muscle">
-          &#128170;
-        </span>
-      </CenteredH1>
+      <FormSpacing>
+        <CenteredH1>
+          Flex your skills!{' '}
+          <span role="img" aria-label="muscle">
+            &#128170;
```

Comment:
Nit pick: could replace with 💪 since it's supported, but just a matter of preference